memgpt/local_llm/settings/settings.py

```diff
@@ -0,0 +1,68 @@
+import json
+import os
+
+from memgpt.constants import MEMGPT_DIR
+from memgpt.local_llm.settings.simple import settings as simple_settings
+from memgpt.local_llm.settings.deterministic_mirostat import settings as det_miro_settings
+
+
+DEFAULT = "simple"
+SETTINGS_FOLDER_NAME = "settings"
+COMPLETION_SETTINGS_FILE_NAME = "completions_api_settings.json"
```

Comment:
@sarahwooders we then look for a file `~/.memgpt/settings/completions_api_settings.json`